dds/DCPS/FilterEvaluator.h

```diff
@@ -92,6 +92,11 @@ class OpenDDS_Dcps_Export FilterEvaluator : public RcObject {
 
   size_t number_parameters() const { return number_parameters_; }
 
+  bool has_unsafe_fields() const
```

Comment:
What is an "unsafe" field, wouldn't it be better to make it "has_key_fields"

Reply:
Probably, I wasn't quite sure what to call it.